pkg/operator/ceph/cluster/register_controllers.go

```diff
@@ -69,6 +70,7 @@ var AddToManagerFuncs = []func(manager.Manager, *clusterd.Context) error{
 	rbd.Add,
 	client.Add,
 	mirror.Add,
+	peermap.Add,
```

Comment:
Why a new controller? Why not just add this in the pool controller since it's already watching for CephBlockPool?

Reply:
A couple  reasons why we need it as a separate controller:
- This is needed for both cephfs and RBD.  There could be race conditions if multiple controllers try to update the configmap.  
- We might have to run this at regular interval of time. For example, if Cluster-1 has the pool but same pool is not created on the peer cluster-2, we need to keep checking in order to  map the pool ID. 

cc @Madhu-1